### ISSUE TITLE
Change Titles on All Pages

### ISSUE THREAD

**Issue by Pkatjs** [Pkatjs/sturdy-octo-barnacle#2]
Site looks great, but it looks like pages other than the index.hrml page have the wrong title.

**Comment by prkd12** [comment]
That's right.. duh . I forget to change the other pages . I'll take care of it right away
